doc/conf.py

```diff
@@ -404,6 +404,7 @@
 		('boards/arm/usb_kw24d512/doc/usb_kw24d512', 'boards/arm/usb_kw24d512/doc/index'),
 		('boards/arm/v2m_beetle/doc/v2m_beetle', 'boards/arm/v2m_beetle/doc/index'),
 		('boards/arm/warp7_m4/doc/warp7_m4', 'boards/arm/warp7_m4/doc/index'),
+		('boards/arm/ruuvi_ruuvitag/doc/ruuvi_ruuvitag', 'boards/arm/ruuvi_ruuvitag/doc/index'),
```

Comment:
This is not needed, since this is is a new board. Remove (unless I am missing something)

Reply:
That's right, you can remove that line.

Reply:
Ok thanks, wasn't sureb about that.

Reply:
I removed that file.